BetaLibanos/BetLib001.xml

```diff
@@ -27,39 +27,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     </msIdentifier>
                               <msContents>
                           <msItem xml:id="ms_i1">
-                              <title ref="LIT4970HagLal"></title>
-                                <msItem xml:id="ms_i1.1">
-                                  <incipit xml:lang="gez">
-                              በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ሥላሴ፡ ዋሕድ፡ ዕሩይ፡ ዘኢይሰደቅ፡ ታሉት፡ ዘኢይነፍድ። ዘሀሎ፡ እምቅድም፡ እምክዋኔሁ፡ እንዘ፡ ኢይትበዓድ፨                       </incipit>
-                                    <explicit xml:lang="gez">
-                          <title/>
-                          </explicit>
-                      </msItem>
-                                    <msItem xml:id="ms_i1.2">
-                                      <title type="complete"/>
-                            </msItem>
-                                  <msItem xml:id="ms_i1.3">
-                                    <title type="complete"/>
-                                </msItem>
-                                <msItem xml:id="ms_i1.4">
-                                  <title type="complete"/>
-                        </msItem>
-                              <msItem xml:id="ms_i1.5">
-                                <title type="complete"/>
-                                </msItem>
-                            <msItem xml:id="ms_i1.6">
-                              <title type="complete"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.7">
-                            <title type="complete"/>
-                        </msItem>
-                        <msItem xml:id="ms_i1.8">
-                        <title/>
-                          <incipit/>
-                        <explicit>  ወከመዝ፡ ነበረ፡ በኵሉ፡ መዋዕለ፡ ሕይወቱ፡ እምአመ፡ ነግሠ፡ እስከ፡ አመ፡ ፈለሰ።  </explicit>
-                      </msItem>
-
-                              <msItem xml:id="ms_i1.9"><locus target="#91ra"/>
+                              <title ref="LIT1470GadlaL"/>
```

Comment:
please here and elsewhere carefully check numbering and nesting

`<msItem>` opens (e.g for the dossier)
 -`<msItem>` for subpart 1 (e.g. gadl)
--`<msItem>` for subsubpart 1 (e. g. introduction)
--`</msItem>` for subsubpart 1 closes
--`<msItem>` for subsubpart 2 (e.g. some other gadl part)
--`</msItem>` for subsubpart 2 closes
-`</msItem>` for subpart 1 closes
 -`<msItem>` for subpart 2 (e.g. miracles)
--`<msItem>` for subsubpart 2-1 (e. g. miracle 1)
--`</msItem>` for subsubpart 2-1 closes
--`<msItem>` for subsubpart 2-2 
--`</msItem>` for subsubpart 2-2 closes
-`</msItem>` for subpart 2 closes

etc
`</msItem>` for item 1 (dossier) closes only after all those nested inside

Reply:
I will check!